docs/requirements.md

```diff
@@ -2,78 +2,48 @@
 
 Running Codacy on a Kubernetes cluster requires the following:
 
--   A Kubernetes 1.14.\* or 1.15.\* cluster provisioned with the [required resources](#cluster-resource-requirements)
--   The [NGINX Ingress Controller](https://github.com/helm/charts/tree/master/stable/nginx-ingress) for Kubernetes
+-   A Kubernetes 1.14.\* or 1.15.\* cluster provisioned with the [required resources](#hardware-requirements)
+-   The [NGINX Ingress Controller](https://github.com/helm/charts/tree/master/stable/nginx-ingress) must be setup in your cluster
 -   A [PostgreSQL server](#postgresql-server-setup) accessible from the Kubernetes cluster
 
-## Cluster resource requirements
+## Hardware Requirements
 
-To have an enjoyable experience with Codacy, you should have the
-following calculations in mind when allocating resources for the
-installation and defining the number of concurrent analysis.
+A good way of understanding how resources are used in Codacy is by understanding how the resources are allocated.
 
-Without accounting for analysis ([next section](#analysis)),
-you should need at least the following resources:
+!["High Level Architecture"](<./images/High Level Architecture - Analysis II.svg> "High Level Architecture")
 
-```text
-CPU: 7 CPU
-Memory: 40 GB
-```
+You can look at Codacy separately as the "Platform" and the "Analysis" cluster. The Platform contains the UI and other
+components important to treat and show results. The Analysis is the swarm of workers that run several linters for
+your project.
 
-Check the
-[values-production.yaml](https://github.com/codacy/chart/blob/master/codacy/values-production.yaml)
-file to find a configuration reference that should work for you to run a
-"production-ready" version of Codacy.
+**The resources needed for Codacy depend a lot on the rate of commits done by your team.**
 
-### Analysis
+Since all of this runs in kubernetes, you can increase the number of replicas in everything, which should give you
+more resilience and throughput, but it will also increase resource usage. A very simplistic overview of the resource
+allocations for each of these is something like this:
```

Comment:
What are "these"? The pods? Or the Codacy components?

We should try to avoid pronouns as much as possible, as this reduces the ambiguity of the provided information.

Reply:
Replaced with 

> A very simplistic overview of the resource allocations for the "Platform" and "Analysis" goes something like this: